<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="isbn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN</string>
    </property>
   </widget>
   <widget class="QLabel" name="category1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="purchasedon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchased On</string>
    </property>
   </widget>
   <widget class="QLabel" name="subcategory1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sub Category</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="authors">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Authors</string>
    </property>
    <widget class="QLabel" name="authorname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Author Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorname_input">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addauthor">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listofauthors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="type">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Type</string>
    </property>
    <widget class="QRadioButton" name="reference">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="textbook">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Book</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="journal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="issuance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Issuance</string>
    </property>
    <widget class="QCheckBox" name="issued_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="issuedon_input">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="issuedon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued On</string>
     </property>
    </widget>
    <widget class="QLabel" name="issuedby">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued By</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="issuedby_input">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="category_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="isbn_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="purchasedon_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_form">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subcategory_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary_Management_System">
    <property name="title">
     <string>Library Management System</string>
    </property>
   </widget>
   <addaction name="menuLibrary_Management_System"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
